src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/PdfGeneration/HealthCertificatePDFGenerationInfoViewModel.swift

```diff
@@ -42,7 +53,36 @@ struct HealthCertificatePDFGenerationInfoViewModel {
 	func generatePDFData(
```

Comment:
There are 3 error cases in this func where the completion is not called. In this case, the UI would wait for the completion forever?

Reply:
i see only one missing completion or fatalError call, after line 66. But in the other cases, a fatalError is thrown. Or do i have 🍅 on my eyes?

Reply:
I have 🍅 all over my face.

Reply:
Sure you want fatalError in case of a network error?

Reply:
You are right, that could be a little bit too hard. I questioned what we should do here, tech spec does not give an answer.